### PR TITLE
Bug - Unit setting in indicator field is ignored

### PR DESCRIPTION
**Pull request by marcin-zapala**
please review in first free time
far from the ideal solution

### CODE REVIEW

File: src/alto-ui/Datagrid/Datagrid.js [hunk 1]
```diff
@@ -45,7 +45,16 @@ const FORMATTERS = {
           .toFormat('dd LLL yyyy')
       : '',
   number: (x, col, row, { locale }) =>
-    formatNumber(x, locale, col.precision || 0, null, col.disableThousandSeparator),
+    formatNumber(
+      col.percent ? x * 100 : x,
```
Comment (marcin-zapala) [inline src/alto-ui/Datagrid/Datagrid.js]: @renaudtertrais i cannot understand this part why i cannot do this in formatNumber? if i manipulate this number in this method i get wrong results when i multiply number here its correct :)

**Comment by marcin-zapala** [comment]
> fix unit test please

improved

File: src/alto-ui/helpers/number.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ const decimalSeparatorByLocale = {
 const DEFAULT_DECIMAL_SEPARATOR = '.';
 
 export const round = precision => n => Math.round(n * 10 ** precision) / 10 ** precision;
+export const percentRound = precision => n => parseFloat(n.toFixed(precision));
```
Comment (renaudtertrais) [inline src/alto-ui/helpers/number.js]:
```js
export const round = precision => n => parseFloat((Math.round(n * 10 ** precision) / 10 ** precision).toFixed(precision));
```

Comment (marcin-zapala) [reply src/alto-ui/helpers/number.js]: Updated
used in a slightly changed form, to dont confuse other devs who jump into this,

**Pull request merged**